src/webpackPlugins/CustomTasksPlugin.js

```diff
@@ -16,23 +17,28 @@ class CustomTasksPlugin {
      * @param {import("webpack").Compiler} compiler
      */
     apply(compiler) {
+        let firstRun = true;
         compiler.hooks.done.tapAsync(this.constructor.name, (stats, callback) => {
```

Comment:
We should change this to tapPromise and make this an async/await function. That'd clean things up a good bit here.